metals/src/main/scala/scala/meta/internal/metals/formatting/IndentOnPaste.scala

```diff
@@ -171,17 +153,7 @@ object IndentOnPaste extends RangeFormatter {
     val overIndent = if (indents.nonEmpty) indents.min else 0
 
     val idented = converted.map(_.reformat(expectedIndent, overIndent, opts))
-
-    // drop leading/trailing empty lines
-    val lastIdx = idented.length - 1
-    val range = 0 to lastIdx
-    val trimmedStart =
-      range.dropWhile(converted(_).isEmpty).headOption.getOrElse(0)
-    val trimmedEnd = range.reverse
-      .dropWhile(converted(_).isEmpty)
-      .headOption
-      .getOrElse(lastIdx) + 1
-    idented.slice(trimmedStart, trimmedEnd)
```

Comment:
There were some issues with this?

Reply:
There is no need to trim the pasted snippet, it's also unexpected behaviour.

Reply:
Ok. I don't remember exactly what was the reason. Probably, I added this just to have less changes in test data :smile: